sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -67,11 +95,17 @@ client = EventGridPublisherClient(endpoint, default_az_credential)
 
 <!-- END SNIPPET -->
 
-#### Looking up the endpoint
+### Looking up the endpoint
+
+#### Event Grid Namespace
+You can find the Namespace endpoint within the Event Grid Namespace resource on the Azure portal. This will look like:
```

Comment:
might be helpful to list which section this will be under. Like overview, etc.

Reply:
I think that would be verbose given that it wasnt added for basic, but will take another pass when merged :)